internal/gh/ghui/push.go

```diff
@@ -453,6 +443,15 @@ func (vm *GitHubPushModel) calculateChangedBranches() tea.Msg {
 			})
 			continue
 		}
+
+		if !isDifferencePRMetadata(avbr, vm) {
```

Comment:
> In addition to hashes, it should check if the PR metadata matches with the local branch metadata. 
https://github.com/aviator-co/av/issues/396

Does this mean that checking the metadata with the hash, together? Or does it check only this case?? 
I implemented it as the latter. But if I'm wrong please let me know, I'll fix it

Reply:
I meant first.